lib/actions/addLinkToProjectAction.js

```diff
@@ -0,0 +1,28 @@
+class AddLinkToProjectAction {
+
+  // validate(){}
+
+  async execute([label, link], project){ 
+    // project.links[label] = link;
+    try {
+      //store data -- > await db.store() if it is async
+      return new AddLinkToProjectResponse(label, link, project)
+    } catch(err) {
+        console.log(err);
+    }
+  }
+}
+
+class AddLinkToProjectResponse {
+  constructor(label, link, project){
+    this.label=label;
```

Comment:
Spaces around that those `=` operators. 

Consider setting up eslint in your editor (and we should get a card for it in the build).